.github/workflows/workflow.yml

```diff
@@ -156,7 +156,9 @@ jobs:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
 
       # Install ocamlfind-secondary and ocaml-secondary-compiler, if needed
-      - run: opam install ./dune.opam --deps-only --with-test
+      - run: |
+          opam pin -n . --with-version=3.20
+          opam install ./dune.opam --deps-only --with-test
```

Comment:
Are you sure this isn't meant to happen later on during `Install deps`? Here we are only building the `dune` package.

Reply:
I'm not sure, but I would imagine it's harmless to do this pin before.